src/components/AuthorCardEditing.vue

```diff
@@ -294,7 +285,7 @@ export default defineComponent({
                 ]
             },
             lastName: {
-                title: 'lastNames',
+                title: 'name particle, family name, name suffix',
```

Comment:
```suggestion
                title: 'name-particle, family-names, name-suffix',
```

Reply:
I also just noticed that the given-names title is `name` and should be `given-names`.